vhdbuilder/scripts/automate_ev2pipeline_trigger.sh

```diff
@@ -5,17 +5,29 @@ source vhdbuilder/scripts/automate_helpers.sh
 
 echo "Triggering ev2 artifact pipeline with Build ID $1"
 
-build_id=$1
-
-set +x
-system_access_token=$2
+build_ids=$1
 
 trigger_pipeline() {
-    az pipelines variable update --name VHD_PIPELINE_RUN_ID --pipeline-id 165939 --value $build_id  # Update the VHD_PIPELINE_RUN_ID with the build ID
-    az pipelines run --id 165939  # 165939 is the pipeline ID for ev2 artifacts pipeline
-}
+    pipeline_id="165939"   # 165939 is the pipeline ID for ev2 artifacts pipeline
+    for build_id in $build_ids; do
+        echo "Build ID for the release is $build_id"
```

Comment:
consider parallelizing these, logging to file, and checking the pids for success like we did for vhd reduction (checkout the script there for how to do it)

Reply:
I didn't parallelize this because we have to update the VHD_PIPELINE_RUN_ID for each build_id, what about a race condition where both processes try to update the variable at once? Since the pipeline takes only 3/4 minutes to run, I left it sequential. Thoughts?